src/material-experimental/mdc-chips/chip-icons.ts

```diff
@@ -84,7 +84,7 @@ const _MatChipRemoveMixinBase:
   CanDisableCtor &
   HasTabIndexCtor &
   typeof MatChipRemoveBase =
-    mixinTabIndex(mixinDisabled(MatChipRemoveBase));
+    mixinTabIndex(mixinDisabled(MatChipRemoveBase), -1);
```

Comment:
There might be a different angle to this issue where the element is actually supposed to be focusable, but the `aria-hidden` was inherited accidentally. If that's the case we should have a different `host` binding that overrides it. @vanessanschmitt is the chip remove button intended to be focusable or not?

Reply:
Hey @crisbeto! I think that matChipRemove icons should not be aria-hidden. tabindex should be -1 for mat-chip-grid remove icons, because the grid is a single tab stop that handles moving focus among the chips and remove icons itself. tabindex should be 0 for mat-chip-set remove icons, because a keyboard user will need to tab to those directly, since mat-chip-set doesn't have arrow key navigation.

Screen reader users won't know about the keyboard bindings and that a chip is removable unless we add an aria-label explaining them, so we could make that aria-label a required Input. But I think it's easier for screen reader users to just use the remove icon. So we could go with the first approach you mentioned, and stop matChipRemove from inheriting aria-hidden from matChipTrailingIcon here: https://github.com/angular/components/blob/49a1324acc05cec1c5ff28d729abfe590f6772dd/src/material-experimental/mdc-chips/chip-icons.ts#L57

Another option is to not make any trailing icons aria-hidden. Some non-matChipRemove trailing icons are purely decorative, and don't provide additional information to the user. For example, these trailing star icons from the demo. Which is why I made trailing icons aria-hidden originally. But we could also unhide them and instead encourage the use of alt text or aria-labels to describe the decorative icons for screen reader users. Whichever you think is best!
![2019-12-27](https://user-images.githubusercontent.com/16886662/71528047-d1e4f200-2892-11ea-8a05-2479c2b392e1.png)

Reply:
Thank you for the context. I've updated it to remove the `aria-hidden`, rather than reset the `tabindex`.